web/context_processors.py

```
@@ -21,3 +21,11 @@ def invitation_notifications(request):
         pending_invites = request.user.received_group_invites.filter(status="pending").count()
         return {"pending_invites_count": pending_invites}
     return {}
+
+
+def available_locales(request):
```

Ruff (0.15.0)

[warning] Unused function argument: `request`

(ARG001)